odoo/models.py

```diff
@@ -3124,18 +3124,34 @@ def fields_get(self, allfields=None, attributes=None):
         has_access = functools.partial(self.check_access_rights, raise_exception=False)
         readonly = not (has_access('write') or has_access('create'))
 
+        translated_attrs = ('string', 'help')
+        translations = {}
+        if self.env.lang:
+            for key in translated_attrs:
+                translations[key] = getattr(self.env['ir.translation'], f'get_field_{key}')(self._name)
```

Comment:
Seriously, metaprogramming for that?  You know, straightforward code quite fast, too :smile: 
```suggestion
        translations = {}
        if self.env.lang:
            translations = {
                'string': self.env['ir.translation'].get_field_string(self._name),
                'help': self.env['ir.translation'].get_field_help(self._name),
            }
```